src/lib/autoinstall/profile_checker.rb

```diff
@@ -0,0 +1,157 @@
+# Copyright (c) [2020] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast2/popup"
+
+require "autoinstall/importer"
+
+Yast.import "AutoInstall"
+Yast.import "AutoinstConfig"
+Yast.import "AutoinstScripts"
+Yast.import "Mode"
+Yast.import "Popup"
+Yast.import "ProfileLocation"
+
+module Y2Autoinstallation
+  class ProfileChecker
```

Comment:
np: It would be nice to have some documentation about this class and its public methods.

Reply:
yes, makes sense